examples/vision/depth_estimation.py

```diff
@@ -28,14 +28,15 @@
 import sys
 
 import tensorflow as tf
```

Comment:
Since this example is TF-only, please add a line to set the backend to TF via `os.environ["KERAS_BACKEND"] = "tensorflow"`

Reply:
Updated to Tensorflow Backend.